apps/balancer-tools/src/app/apr/(utils)/fetchPoolSnapshots.ts

```diff
@@ -0,0 +1,43 @@
+import { PoolSnapshotInRangeQuery } from "@bleu-balancer-tools/gql/src/balancer/__generated__/Ethereum.server";
+
+import { pools, poolsWithoutCache } from "#/lib/gql/server";
+
+import { dateToEpoch, generateDateRange } from "../api/(utils)/date";
+
+export async function fetchPoolSnapshots(
+  to: number,
+  extendedFrom: number,
+  network: string,
+  poolId: string,
+): Promise<PoolSnapshotInRangeQuery> {
+  let res: PoolSnapshotInRangeQuery;
+
+  const now = new Date();
+  const currentDateUTC = dateToEpoch(
```

Comment:
if we were using the solution described by https://github.com/bleu-fi/balancer-tools/pull/430/files#r1365501152 we wouldn't need to do this since all UNIX timestamps are in UTC time

Reply:
not sure if I follow, even using https://github.com/bleu-fi/balancer-tools/pull/430/files#r1365501152 it would be needed to get `utcMidnightTimestampOfCurrentDay`